src/connections/ResetPassword.ts

```diff
@@ -0,0 +1,61 @@
+import { createSelector } from "reselect";
+
+import {
+  requestPasswordResetFetchFailed,
+  requestPasswordResetIsFetching,
+  resetPasswordFetchFailed,
+  resetPasswordIsFetching
+} from "@/generated/v3/userService/userServicePredicates";
+import { ApiDataStore, PendingErrorState } from "@/store/apiSlice";
+import { Connection } from "@/types/connection";
+import { connectionHook } from "@/utils/connectionShortcuts";
+
+export const selectResetPassword = (store: ApiDataStore) => Object.values(store.logins)?.[0]?.attributes;
+
+type RequestResetPasswordConnection = {
+  isLoading: boolean;
+  requestFailed: PendingErrorState | null;
+  isSuccess: boolean;
+  requestEmail: string;
+};
+
+const requestPasswordConnection: Connection<RequestResetPasswordConnection> = {
+  selector: createSelector(
+    [requestPasswordResetIsFetching, requestPasswordResetFetchFailed, selectResetPassword],
+    (isLoading, requestFailed, selector) => {
+      return {
+        isLoading: isLoading,
+        requestFailed: requestFailed,
+        isSuccess: (selector as any) != null,
```

Comment:
Why is `as any` needed here? The FE codebase isn't very well typed in a lot of cases so some `any`s are to be expected, but I've put a lot of work into making sure we can start to fix that with v3. I wouldn't expect an `any` anywhere in Connection code.